functional/tests/core-discovery-jsonrpc.test.mjs

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+import { expect, jest, test, afterAll, beforeAll } from "@jest/globals";
+import * as utilities from "./utilities.mjs";
+
+jest.setTimeout(30030);
+
+const httpPort = 3335;
+const socketPort = 9990;
+
+beforeAll(async () => {
+  console.log(
+    await utilities.mfState(
```

Comment:
store response in const variable and validate it.

Reply:
We can't validate it as it's inside the beforeAll block. If we create test inside that jest will throw error.